This file is part of mingw-cross-env.
See doc/index.html for further information.

This patch taken from:
https://gitorious.org/openscenegraph/osg/commit/c535cf1edc2751d02fbdb9e7fef035cb797dff97?format=patch

From c535cf1edc2751d02fbdb9e7fef035cb797dff97 Mon Sep 17 00:00:00 2001
From: Robert OSFIELD <robert.osfield@gmail.com>
Date: Mon, 18 Jul 2011 09:40:58 +0000
Subject: [PATCH] Changed the HAVE_CONFIG_H guard so that it uses #ifndef _MSC_VER rather than _WIN32 to avoid problems with Mingw and Cygwin builds of DCMTK.

---
 src/osgPlugins/dicom/ReaderWriterDICOM.cpp |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/osgPlugins/dicom/ReaderWriterDICOM.cpp b/src/osgPlugins/dicom/ReaderWriterDICOM.cpp
index 785ac9e..500cb7e 100644
--- a/src/osgPlugins/dicom/ReaderWriterDICOM.cpp
+++ b/src/osgPlugins/dicom/ReaderWriterDICOM.cpp
@@ -19,7 +19,7 @@
 #include <osgVolume/RayTracedTechnique>
 
 #ifdef  USE_DCMTK
-    #ifndef _WIN32
+    #ifndef _MSC_VER
         #define HAVE_CONFIG_H
     #endif
 
-- 
1.6.1
